charts/README.md

```diff
@@ -0,0 +1,71 @@
+# EdgelessDB helm charts
+
+## Documentation
+
+See the [Getting Started Guide](https://docs.edgeless.systems/edgelessdb/#/getting-started/quickstart-sgx) to set up a EdgelessDB in a few simple steps.
```

Comment:
Not sure if it makes sense to refer to the quickstart here as it doesn't cover k8s deployment.